respy/solve.py

```diff
@@ -52,7 +52,22 @@ def get_solve_func(params, options):
 
 
 def solve(params, options, state_space):
-    """Solve the model."""
+    """Solve the model.
+
+    Parameters
+    ----------
+    params : pandas.DataFrame
+        DataFrame containing parameter series.
+    options : dict [str, int or list or dict]
+        Dictionary containing model attributes.
+    state_space : :class:`~respy.state_space.StateSpace`
+        Unsolved state space of the model.
+
+    Returns
+    -------
+    state_space : :class:`~respy.state_space.StateSpace`
+        Solved state space of the model.
+    """
     optim_paras, options = process_params_and_options(params, options)
```

Comment:
Not sure if this is a stupid question, but why is this step in `get_solve_func` and in `solve`, wouldn't it suffice to do it once in `get_solve_func`? (this is unrelated to the doctstrings, I just stumbled over it)